pkg/vere/vere.h

```diff
@@ -321,6 +321,7 @@
 
         c3_o    beb;                        //  --behn-allow-blocked
         c3_z    siz_i;                      //  --lmdb-map-size
+        c3_o    wat_o;                      //  --watch-replay
```

Comment:
This should either be removed, or integrated into `_cw_play_fork()` (controlling whether `--watch-replay` is set). For the latter, I'd make it default on and have the flag disable the watcher thread.